src/main/java/org/mule/extension/introspection/ExtensionModel.java

```diff
@@ -36,7 +36,7 @@
  *
  * @since 1.0
  */
-public interface ExtensionModel extends Described, Capable
+public interface ExtensionModel extends Described, Capable, ExtensibleModel
```

Comment:
My original name was ExtendedModel but I had concerns about it and changed it to Extensible as the best my mind could produce. Do you have any suggestions?